js/currencycom.js

```diff
@@ -483,8 +483,8 @@ module.exports = class currencycom extends Exchange {
                 // https://github.com/ccxt/ccxt/issues/4286
                 // therefore limits['price']['max'] doesn't have any meaningful value except undefined
                 limitPriceMin = this.safeNumber (filter, 'minPrice');
-                const maxPrice = this.safeNumber (filter, 'maxPrice');
-                if ((maxPrice !== undefined) && (maxPrice > 0)) {
+                const maxPrice = this.safeString (filter, 'maxPrice');
+                if ((maxPrice !== undefined) && (Precise.stringLt (maxPrice, '0'))) {
```

Comment:
We will have to add parseNumber below before returning this to the caller.

Reply:
@samgermain shouldn't we use `stringGt` (>0) instead?

Reply:
> We will have to add parseNumber below before returning this to the caller.

It already is on line 560